### PR TITLE
openapi: fix stdcm types

### LINKED ISSUE #4990: front: fix stdcm types in openApi.yaml

### PR DESCRIPTION
**Pull request by clarani**
closes #4990

### CODE REVIEW

File: front/src/applications/operationalStudies/consts.ts [hunk 1]
```diff
@@ -100,14 +100,21 @@ export const STDCM_MODES = Object.freeze({
...
-});
+export enum STDCM_REQUEST_STATUS {
+  IDLE = 'IDLE',
+  PENDING = 'PENDING',
+  SUCCESS = 'SUCCESS',
+  REJECTED = 'REJECTED',
+  CANCELED = 'CANCELED',
+  NO_RESULT = 'NO_RESULT',
+}
+export type StdcmRequestStatus =
```
Comment (anisometropie) [inline front/src/applications/operationalStudies/consts.ts]: use `STDCM_REQUEST_STATUS` as a type

**Review by anisometropie — DISMISSED** [review]
this needs to be merged after #4824

**Comment by clarani** [comment]
This must be merge after #4822

**Review by flomonster — APPROVED** [review]
Reviewed editoast files

**Review by flomonster — APPROVED** [review]

**Pull request merged**